### ISSUE TITLE
Generated images aren't saved in the usual Automatic1111 paths

### ISSUE THREAD

**Issue by curtwagner1984** [isekaidev/stable.art#25]
Hello, this is a great plug it, but from my testing, generated images aren't saved in the automatic1111 output folder. 
I would very much like this option because it's easy to come back to those output images and read out the prompt and model information. 

Of course, it would be even better if the Photoshop UI had access to the automatic1111 output folder(just like the 'explore' tab, but for the local automatic1111 output folder.) the so that previously generated images could be used. But for now, I'll just be happy if all generated images are saved in the automatic1111 folder.

(It would be even better if a specific subfolder would be created for images generated using the plugin in the automatic1111 output folder)

**Comment by 4lt3r3go** [comment]
i would also know where TEMP images are stored

**Comment by Woisek** [comment]
I also think this would be an important thing ...

**Comment by HustLion** [comment]
> Hello, this is a great plug it, but from my testing, generated images aren't saved in the automatic1111 output folder. I would very much like this option because it's easy to come back to those output images and read out the prompt and model information.
> 
> Of course, it would be even better if the Photoshop UI had access to the automatic1111 output folder(just like the 'explore' tab, but for the local automatic1111 output folder.) the so that previously generated images could be used. But for now, I'll just be happy if all generated images are saved in the automatic1111 folder.
> 
> (It would be even better if a specific subfolder would be created for images generated using the plugin in the automatic1111 output folder)

I also think this would be really useful. Hoping for this to be implemented.

**Comment by isekaidev** [comment]
Unfortunately, Photoshop API limits file access, so the plugin cannot save files to the A1111 folder but can save images to its own local folder. I added an option to enable saving all generated images and also added a link that opens a folder with all images (simply click it and it will open Finder/File Explorer).

Access to generated images inside the plugin UI is also in the plans but will be added later.

<img width="599" alt="image" src="https://user-images.githubusercontent.com/118139897/216810415-860568b8-8c08-403c-9a8b-1229ac5264e1.png">

**Comment by curtwagner1984** [comment]
Oh, I see. So you're saying that automatic needs a feature to continue
savings generated images to folders even if it was generated using an API
call and not from automatic itself, right?

**Comment by isekaidev** [comment]
Yes, Automatic does not save images generated with API, so the plugin itself manages this.

**Comment by curtwagner1984** [comment]
Ok thanks, i think it's sensible for automatic to have an option to save an
image even if it was generated with the api.

**Comment by screan** [comment]
> Unfortunately, Photoshop API limits file access, so the plugin cannot save files to the A1111 folder but can save images to its own local folder. I added an option to enable saving all generated images and also added a link that opens a folder with all images (simply click it and it will open Finder/File Explorer).
> 
> Access to generated images inside the plugin UI is also in the plans but will be added later.
> 
> <img alt="image" width="599" src="https://user-images.githubusercontent.com/118139897/216810415-860568b8-8c08-403c-9a8b-1229ac5264e1.png">

does this work? i set the folder but seeing no images.
